umbral-pre-wasm/Cargo.toml

```diff
@@ -21,4 +21,6 @@ wee_alloc = "0.4"
 [package.metadata.wasm-pack.profile.release]
 # See https://github.com/rustwasm/wasm-pack/issues/886
 # Maybe at some point in time this won't be necessary.
-wasm-opt = ["-Oz", "--enable-mutable-globals"]
+# wasm-opt = ["-Oz", "--enable-mutable-globals"]
+# TODO: wasm-opt fails with error "Error: failed to execute `wasm-opt`: exited with signal: 11"
```

Comment:
That's strange. I thought I had some outdated package for which it worked, but I updated every dependency, and `wasm-pack` to 0.10, and these options still work for me.

But I think we can remove this whole section completely - https://github.com/rustwasm/wasm-pack/issues/886 has been fixed starting from `wasm-bindgen=0.2.70`